README.md

```diff
@@ -31,6 +31,14 @@ It allows users to:
 
    Apps directory is set using the `WEB_ASSET_APPS_PATH` environment variable.
 
+### Installating Md-Viewer to the oCIS Deployment
+
+1. Navigate to the `deployments/examples/ocis_full/web_extensions` folder of your installation and copy [`mdviewer.yml`] into the [`web_extensions`](https://github.com/owncloud/ocis/tree/master/deployments/examples/ocis_full/web_extensions) subfolder.
+
+2. Add `MDVIEWER=:web_extensions/mdviewer.yml` to the `## oCIS Web Extensions ##` section of the `.env` file of your installation (file is located in `deployments/examples/ocis_full`).\ Add that variable in the `COMPOSE_FILE` variable at the last line.
```

Comment:
```suggestion
2. Append `:web_extensions/mdpresentation-viewer.yaml` to `COMPISE_FILE` variable in the [`.env`](https://github.com/owncloud/ocis/tree/master/deployments/examples/ocis_full/.env)
    ```env
    COMPOSE_FILE=docker-compose.yml${...}:web_extensions/mdpresentation-viewer.yaml
    ```
```

Reply:
It's possible to do it this way without defining a variable for `MDVIEWER` (or MDPRESENTATIONVIEWER), but I would recommend to do it in the same way as the other extensions are added. Although this means one more line of code, it's a bit less risky that someone messes up with the syntax.